modulemd/include/modulemd-2.0/modulemd-errors.h

```diff
@@ -78,7 +195,10 @@ typedef enum
  * encountered while parsing a YAML document.
  *
  * Since: 2.0
+ * Deprecated: 2.9
+ * Use #ModulemdYamlError instead.
```

Comment:
As noted above...

A single line "Deprecated: 2.9: Use #ModulemdYamlError instead."

And `MMD_DEPRECATED_TYPE_FOR (ModulemdYamlError)` below.